test/src/receiver/main.cpp

```diff
@@ -96,10 +96,10 @@ cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp)};
 cadmium::dynamic::modeling::Models submodels_TOP = {generator, receiver1};
 cadmium::dynamic::modeling::EICs eics_TOP = {};
 cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<Receiver_defs::out,outp>("receiver1")
+  cadmium::dynamic::translate::make_EOC<receiver_defs::out,outp>("receiver1")
 };
 cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Receiver_defs::in>("generator","receiver1")
+  cadmium::dynamic::translate::make_IC<iestream_input_defs<message_t>::out,receiver_defs::in>("generator","receiver1")
```

Comment:
line length > 80. 
Please break into smaller lines.
Many of the lines in your test files have length>80. Please check and correct.

Reply:
Changes to the main.cpp has been done. Lines updated with this commit is mentioned in the commit description.
Thank You.